src/js/pensions/config/form.js

```diff
@@ -1070,13 +1070,25 @@ const formConfig = {
           title: item => `${item.veteranFullName.first} ${item.veteranFullName.last} expenses`,
           schema: {
             type: 'object',
+            required: ['view:hasOtherExpenses'],
             properties: {
+              'view:hasOtherExpenses': {
+                type: 'boolean'
+              },
               otherExpenses
             }
           },
           uiSchema: {
             'ui:title': createHouseholdMemberTitle('veteranFullName', 'Medical, legal, or other unreimbursed expenses'),
-            otherExpenses: otherExpensesUI
+            'view:hasOtherExpenses': {
+              'ui:title': 'Do you have any medical, legal or other unreimbursed expenses? (You will need to supply receipts)',
```

Comment:
Now you've got me wondering whether this belongs in a warning like burial transportation receipts. @goldenmeanie would that make sense here?